src/components/activity/ActivityFormPanel.svelte

```diff
@@ -67,6 +69,12 @@
   $: hasChildren = child_ids ? child_ids.length > 0 : false;
   $: isChild = parent_id !== null;
   $: startTimeField = field<string>(startTime, [required, timestamp]);
+  $: if (duration) {
+    const startTimeISO = new Date(getUnixEpochTime(startTime)).toISOString();
```

Comment:
Let me know if there's a cleaner way to manipulate this Activity DOY start time string, maybe I missed some other pattern or utility. Can also move this into a util if you think that'd be useful.